lib/dealership.rb

```diff
@@ -0,0 +1,51 @@
+class Dealership
+
+  attr_reader :name, :address
+
+  def initialize(name, address)
+    @name = name
+    @address = address
+    @inventory = []
+  end
+
+  def inventory
+    @inventory
+  end
+
+  def inventory_count
+     inventory.length 
+  end
+
+  def add_car(car)
+    @inventory << car
+  end
+
+  def has_inventory?
+    if @inventory == []
+     return false
+    else
+      true
+    end
+  end
+
+  def cars_by_make(name)
+  inventory.select do |car|
+     car.make == name
+    end
+  end
+
+  def total_value
+  individual_car_cost = []
+  inventory.each do |car|
+    individual_car_cost << car.total_cost
+  end
+  individual_car_cost.sum
+  end
+
+  def details
+    details = {
+      "total_value" => total_value,
+      "address" => address
+    }
```

Comment:
```suggestion
    {
      "total_value" => total_value,
      "address" => address
    }
```

You can actually just straight up write the hash. No need to set it equal to a variable since we never call that variable anywhere.